File: Codeforces/4/A_Watermelon.go
// Problem: A - Watermelon
// Contest: 4
// Language: Go
// Tags: brute force, math, *800

package main

import (
	"fmt"
)

func main() {
	var w int

	// Take integer input
	fmt.Scan(&w)

	// Check the condition
	if w > 2 && w%2 == 0 {
		fmt.Println("YES")
	} else {
		fmt.Println("NO")
	}
}

File: Codeforces/4/A_Watermelon.cpp
// Problem: A - Watermelon
// Contest: 4
// Language: C++20 (GCC 13-64)
// Tags: brute force, math, *800

#include <iostream>
using namespace std;

int main() {
    int w;
    cin >> w;

    if (w > 2 && w % 2 == 0)
        cout << "YES";
    else
        cout << "NO";

    return 0;
}